src/storage/src/hummock/store/version.rs

```diff
@@ -466,7 +469,10 @@ impl HummockVersionReader {
                         .await?
                         {
                             // todo add global stat to report
-                            local_stats.report(self.stats.as_ref());
+                            local_stats.report(
+                                self.state_store_metrics.as_ref(),
+                                Some(read_options.table_id),
```

Comment:
Shall we let `report` take a table id str to avoid multiple to_string for the same table_id?